### PR TITLE
Feature/issue 37/add donation as a donor of a specific organization

### PR DESCRIPTION
**Pull request by arsalananwar11**
# [Issue #37](link-to-issue)

## Description
- Add donation functionality added
- [Refactor] Separated CSS and JS files and stored them in static folder
- [Refactor] Restructured the forms in donor dashboard so that multiple forms can be accommodated
- [Test] Updated tests for recipient dashboard
- [Bug Fix] Donor Organization Link is now fixed on the recipient dashboard

## Change Type (delete non-relevant options)
- 🪲 Bug fix (non-breaking change which fixes an issue)
- 💡 New feature (non-breaking change which adds functionality)
- 🧪 Test (for adding tests)
- 🔄 Refactor (for code refactoring and tech debt)

## Dependencies

- This PR is bound to have merge conflicts with the PRs raised prior to this.

### CODE REVIEW

**Comment by arsalananwar11** [comment]
image

image

File: src/recipient_dashboard/templates/recipient_dashboard/dashboard.html [hunk 1]
```diff
@@ -29,9 +29,9 @@ <h2 class="subtitle mt-3">List of Available Donations:</h2>
                                                href="https://www.google.com/maps/dir/?api=1&destination={{ donation.organization_id.address|urlencode }}"
                                                target="_blank"
                                                rel="noopener noreferrer">
-                                                {{ donation.organization_id.organization_name }}
+                                                {{ donation.organization.organization_name }}
```
Comment (maxgrove42) [inline src/recipient_dashboard/templates/recipient_dashboard/dashboard.html:32]: FYI (no action required) - I don't think this will have merge conflicts, but I think we had talked about me handling this (and I opened a PR for it here: https://github.com/gcivil-nyu-org/wed-fall24-team5/pull/103) ! 

IMO this change should probably be in its own PR just to keep the scope of a PR centered

File: src/recipient_dashboard/tests.py [hunk 2]
```diff
@@ -27,14 +27,14 @@ def setUp(self):
 
         # Create active donations for testing
         self.donation1 = Donation.objects.create(
-            organization_id=self.organization,
+            organization_id=self.organization.organization_id,
```
Comment (maxgrove42) [inline src/recipient_dashboard/tests.py:30]: Nit here - Same thing here, we probably want this to be in a separate PR in the future !

**Review by maxgrove42 — APPROVED** [review]
Really nice work ! looks clean!

File: src/recipient_dashboard/templates/recipient_dashboard/dashboard.html (hunk 1 above)
Comment (arsalananwar11) [reply src/recipient_dashboard/templates/recipient_dashboard/dashboard.html:32]: Ahh cool, totally agree to your point!

File: src/recipient_dashboard/tests.py (hunk 2 above)
Comment (arsalananwar11) [reply src/recipient_dashboard/tests.py:30]: Sure Max, we need objective reviews like these!
